r_basics_visualize_data/r_basics_visualize_data.md

```diff
@@ -0,0 +1,1063 @@
+<!--
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  0.4
+language: en
+narrator: US English Female
+title: R Basics: Visualizing Data With ggplot2
+comment:  Learn how to visualize data using R's `ggplot2` package.
+long_description: Do you want to learn how to make some basic data visualizations (graphs) in R?  In this module you'll learn about the "grammar of graphics" and the base code that you need to get started.  We'll use the basic ingredients of a tidy data frame, a geometric type, and some aesthetic mappings (we'll explain what all of those are).  This module teaches the use of the `ggplot2` package, which is part of the `tidyverse` suite of packages.
+
+@learning_objectives  
+
+After completion of this module, learners will be able to:
+
+- Describe what makes a data frame "tidy"
```

Comment:
I do think that tidy data is very important in both transform and data viz and it might make sense to pull together all the tidy data info together into another module.  That would also make both the transform and viz more focused and shorter.  Curious to hear your take

Reply:
oooh, I like that idea. we're also very likely going to need to refer to the concept of tidy data in many other modules, and it's an important concept that doesn't just apply to the world of R (even if it's a term more frequently utilized in the R/tidyverse conversations)  -- it makes sense to pull that out as its own unit of content imo